### ISSUE TITLE
Don't install epel-repository if already published via Spacewalk

### ISSUE THREAD

**Issue by epleterte** [HariSekhon/nagios-plugins#170]
In my environment the EPEL repository is published via Spacewalk, so installing the `epel-release`-package is not necessary.
My initial workaround was to comment out the `yum install` lines from the _Makefile_, but it's listed in _setup/rpm-packages.txt_ and got installed anyway (so I had to comment out there too). 

This is probably not a high priority thing, but it would be nice to avoid duplicate repo sources.
I'm guessing there is a similar case for people using Pulp/Candlepin (possibly via Katello, the Spacewalk successor).

An easy test to check if the EPEL repository is available is this (the `-i` flag is not necessary for me, but may be good for someone else?):
`spacewalk-channel --list | grep -i epel`

**Comment by HariSekhon** [comment]
@epleterte 

Thanks very much for raising this... it'll be a while before I can do enough testing on this as I'm constantly travelling right now and do not have access to my private Spacewalk infrastructure from here at the moment, but definitely something I will come back to try to improve in future.

You're welcome to submit a patch pull request before then if you wish.

Thanks

Hari

**Comment by epleterte** [comment]
Sure. Is there any reason to have it listed in both the _Makefile_ and _setup/rpm-packages.txt_? It doesn't seem so? :)

**Comment by HariSekhon** [comment]
Yes there is - `setup/rpm-packages.txt` is a vanilla try and something that can be run by hand separately, whereas the one in `Makefile` is a more compatible that will determine and download the correct rpm directly if a straight yum install doesn't work - this will succeed on older versions of RHEL / CentOS as well and is left there for compatibility even after I abstracted all the packages for different OSs out to `setup/`.

**Comment by HariSekhon** [comment]
I've just added a comment to the `Makefile` to make this more obvious.
